.compute

```diff
@@ -2,12 +2,14 @@
 
 set -xe
 
-apt-get install -y python3-venv
+apt-get install -y python3-venv libopus0
```

Comment:
Why isn't libopus0 in requirements.txt?

Reply:
This is the system-level Opus library which is required by the Python package.
The wrapping module in `requirements.txt`:
```opuslib == 2.0.0```

Reply:
Oh I'm dumb I was reading this is a pip install.